supergloo/docs/mesh/tutorials-2-root-cert.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Tutorial: Configuring a Custom Certificate Authority"
+weight: 4
+---
+
+### Summary
+
+In this tutorial we'll take a look at how to set a custom certificate authority (CA) which will issue the 
+certificates used for mutual TLS using SuperGloo.
+
+When mutual TLS is enabled between services, client and server certificates are generated for each service 
+by the mesh which manages them. By default those certificates are created using a self-signed root certificate and key generated by 
+the mesh, which acts as certificate authority for the services in the mesh. 
+
+Using SuperGloo, we can generate certs for use in our mesh using root certificates created by an external CA. This tutorial 
+will walk through the steps required to set the root certs used for mTLS in Istio. 
+
+### Tutorial
+
+Let's demonstrate the ability to rotate the root certificates used by Istio. First we'll rotate the certs, then we'll
+verify that the pods in our mesh were updated by directly inspecting the certificates loaded by one of our sidecar proxies.
+
+First, ensure you've:
+
+- [installed SuperGloo with mTLS enabled](../../installation)
```

Comment:
Capitalize the "I' 
Also.. don't you mean "Install SuperGloo"? and add the "with mTLS enabled" to the next line (about Istio installation?)

Reply:
good point